src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -3689,10 +3690,16 @@ internal Collection<PSObject> NewItem(string[] paths, string name, string type,
 
             foreach (string path in paths)
             {
+                string resolvePath = null;
                 if (path == null)
                 {
                     PSTraceSource.NewArgumentNullException("paths");
                 }
+                else
+                {
+                    // To be compatible with Linux OS.
+                    resolvePath = path.TrimEnd(Path.DirectorySeparatorChar).TrimEnd(Path.AltDirectorySeparatorChar);
```

Comment:
>resolvePath [](start = 20, length = 11)

I'm not sure that this can be so simple, are we guaranteed that the order will be this? For example, will we'll always get "\", and then "/"?